### DISCUSSION TITLE
Cant auth in CMD funi.js HELP PLEASE!!!!!

### DISCUSSION THREAD

**Discussion by yagmur888** [anidl/multi-downloader-nx#520] [Q&A]
Can't login via command line in auto multidownloader
![Screenshot_9](https://github.com/anidl/multi-downloader-nx/assets/118109825/03c45e87-3a70-4a65-990c-e91720240577)

**Comment by Jaynator495** [comment]
If you compiled it, it would be in a folder called "lib", and you would need to `cd` into the correct directory.
Alternatively, you can get the GUI version from the releases tab if you want something easier.
